plutus-conformance/test-cases/uplc/evaluation/builtin/semantics/replicateByte/case-07/case-07.uplc.budget.expected

```diff
@@ -1,2 +1,2 @@
-({cpu: 260294
-| mem: 601})
\ No newline at end of file
+({cpu: 260453
```

Comment:
Why have all these budgets changed?  These are very simple  textual UPLC programs and I don't see why the changes in this PR would have affected their execution costs.

Reply:
I tried to figure that out but couldn't understand the tests. There is TH involved which makes it harder to reason statically. How can I see those UPLC programs and what has changed in them?

Reply:
Nothing's changed!  The conformance test machinery [traverses the directory hierarchy](https://github.com/IntersectMBO/plutus/blob/e0b8db206b16f296529f5b45939523f4683f0f2e/plutus-conformance/src/PlutusConformance/Common.hs#L72) looking for directories with no subdirectories and when it finds one it looks for a `.uplc` file and if it finds one it loads and executes it and then checks the result and budget against golden files.  Look inside the directories [here](https://github.com/IntersectMBO/plutus/tree/master/plutus-conformance/test-cases/uplc/evaluation/builtin/semantics/replicateByte) for example.  The input files haven't changed, so the results shouldn't change either.  The only thing that I can think of is that I messed something up when I renamed the files.  I'll see if I can work out what's going on.

Reply:
@Unisay I'm not sure what's happened, but you seem to have checked in golden files with incorrect CPU budgets but correct memory budgets.  I checked out a completely new copy of the repository and ran `cabal test  haskell-conformance --test-option "-p /writeBits/" and the tests all failed and said that the budget results should have been the old versions (ie,what's in `master`).  Running the test files manually with `uplc -c` also returns the budgets that are in `master`.

Reply:
They fail locally and `--accept` flag doesn't seem to update golden files.

<img width="558" alt="image" src="https://github.com/user-attachments/assets/ff27fcfc-b783-40c1-ba6c-681b9f23d44f">

Reply:
What does `unexpected success` mean in the context of comparing execution budgets?

Reply:
Ok, I've reverted all the golden file updated uner the `agda-conformance` and tests pass now (locally)

Reply:
> What does `unexpected success` mean in the context of comparing execution budgets?

Some tests are expected to fail, and "unexpected success" means that they haven't.  

After some digginng it turns out that this comes from [Test.Tasty.ExpectedFailure](https://hackage.haskell.org/package/tasty-expected-failure-0.12.3/docs/Test-Tasty-ExpectedFailure.html).